frontend/src/features/auth/components/SignUpComplete.tsx

```
@@ -0,0 +1,46 @@
+
+const SignUpComplete = () => {
+  const navigate = useNavigate();
+
+  return (
+    <div className='relative flex h-screen w-full flex-col items-center justify-center select-none'>
+      <div className='absolute -z-10'>
+        <SI.SignUpBackground />
+      </div>
+      <div className='z-10 flex flex-col items-center'>
+        <div className='mb-8'>
+          <img src={FolderIcon} />
```

Biome (2.3.13)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)